cmd/daprd/components/state_etcd.go

```diff
@@ -21,5 +21,7 @@ import (
 )
 
 func init() {
-	stateLoader.DefaultRegistry.RegisterComponent(etcd.NewEtcdStateStore, "etcd")
+	stateLoader.DefaultRegistry.RegisterComponent(etcd.NewEtcdStateStoreV1, "etcd")
```

Comment:
Rather than doing this, should change the component registry to default to `v1` if omitted

Reply:
Done

Reply:
There is now a mechanism for defining versioning behavior of component types which have multiple version.